send/ad_group_mu_ucn

```diff
@@ -214,3 +202,97 @@ sub process_update() {
 	}
 
 }
+
+sub add_members_to_entry {
+
+	my $ad_entry = shift;
+	my $to_be_added = shift;
+	my $return_code = 1;
```

Comment:
This is correct, but I would prefer something like:
```
our $SUCCESS = 1;
our $FAIL = 0;

my $return_code = $SUCCESS;
```
The same in all places where you are using these return states.

Reply:
Done